rllib/core/rl_module/marl_module.py

```diff
@@ -430,6 +500,26 @@ def _check_before_build(self):
                 "SingleAgentRLModuleSpecs for each individual module."
             )
 
+    def to_dict(self):
```

Comment:
I know it's obvious but this one is missing a return type!

Reply:
:) done